prod/README.md

```diff
@@ -0,0 +1,84 @@
+# Production Deployment
+This directory contains everything required to deploy MIQA in production using docker-compose.
+
+# Instructions
+All of these commands should be run from the `prod` directory.
+
+### Set the variables in .env
+The `.env` file contains some default values for local testing that will need to overwritten for production deployments.
+For the build and setup process these values are mostly placeholders, but they will need to be set correctly before the site is made public.
+
+`DJANGO_EMAIL_URL` must be of the form `submission://username:password@my.smtp.server:port`. See the [documentation](https://github.com/migonzalvar/dj-email-url#supported-backends) for details with encoding special characters.
+
+You can also specify those variables explicitly in your local shell environment instead of editing the file.
+`docker-compose` will check for local shell values before defaulting to the values in `.env`.
+
+### Build the docker image
+```
+docker-compose build
+```
+
+### Apply the DB migrations
+Until you do this, the postgres DB is completely empty.
+This will not populate any data, only set up tables.
+```
+docker-compose run django ./manage.py migrate
+```
+
+### Create a superuser to administer the server
+**IMPORTANT**: Set both the Username and Email to the same valid email address.
+Admin logins will break if you do not.
+```
+docker-compose run django ./manage.py createsuperuser
+```
+
+### Run the server
+```
+docker-compose up
+```
+
+### Set up OAuth Application
+The app should now be running and accessible in your browser.
```

Comment:
I'd like to add an idempotent manage.py command to set up the OAuth application with all these knobs hard-coded to the right value.